### PR TITLE
Happy New Year

### PR DESCRIPTION
**Pull request by whyman**

### CODE REVIEW

File: src/config/config.h [hunk 1]
```diff
@@ -4,7 +4,7 @@
     
     config.h - this file is part of Gerbera.
     
-    Copyright (C) 2020-2020 Gerbera Contributors
+    Copyright (C) 2021-2021 Gerbera Contributors
```
Comment (KarlStraussberger) [inline src/config/config.h:7]: Upps: 2021-2021

File: src/config/client_config.cc [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
     client_config.cc - this file is part of Gerbera.
 
-    Copyright (C) 2020 Gerbera Contributors
+    Copyright (C) 2021 Gerbera Contributors
```
Comment (KarlStraussberger) [inline src/config/client_config.cc:7]: Wouldn't it be better to have 2020-2021?

**Review by KarlStraussberger — COMMENTED** [review]
Wouldn't it be better to have 2020-2021 instead of 2021 only?

File: src/config/client_config.cc (hunk 2 above)
Comment (whyman) [reply src/config/client_config.cc:7]: Yes, this was just me lazily find and replacing!

Comment (KarlStraussberger) [reply src/config/client_config.cc]: I managed to break bitmap images in this way (replace in all files)

**Pull request merged**